### PR TITLE
New 'avatarIsNoUser' item field passed to avatar as 'isNoUser'

### PR DESCRIPTION
**Pull request by eneiluj**
Hey,

As Avatar's `isNoUser` prop is now false by default, we can't display colored initials for non user stuff in WidgetItem default Avatar template. The `avatarIsNoUser` item data field is passed to a WidgetItem prop which just forwards it to the Avatar component in default Avatar template.

### CODE REVIEW

**Comment by juliushaertl** [comment]
> As Avatar's isNoUser prop is now false by default

Did that change? I'm generally fine with the change, but it is a breaking one I'd say so we should make sure to bump the major and properly add it to the changelog.

**Comment by eneiluj** [comment]
Well my statement is wrong. It's been a long time Avatar's `isNoUser` prop is false by default but at some point (v2.8.0 I think), it stopped displaying initials as fallback content when provided user name is not a user. Now, `isNoUser` must be true for initials to be displayed in this case (which makes sense :grin:).

So this PR's change will require changes for all use cases where "real users" avatar should be displayed.

It might make more sense to set `avatarIsNoUser` 's default to false so it sticks to `isNoUser` 's default and does not affect people displaying "real users". What do you think?

**Comment by eneiluj** [comment]
> we should make sure to bump the major and properly add it to the changelog

:+1:

**Comment by eneiluj** [comment]
Rebased, changed the default value of `avatarIsNoUser` to false, added changelog so we can write it progressively.

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Change Log
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/)
+and this project adheres to [Semantic Versioning](http://semver.org/).
+
+## 1.1.0 – 2020-11-13
```
Comment (juliushaertl) [inline CHANGELOG.md:7]:
```suggestion
## 1.1.0 – unreleased
```

**Review by juliushaertl — APPROVED** [review]

**Pull request merged**